src/handlers/comment/handlers/payout.ts

```diff
@@ -27,7 +27,7 @@ export const payout = async (body: string) => {
   }
 
   const IssueComments = await getAllIssueComments(issue.number);
-  if (IssueComments === null) {
+  if (IssueComments.length === 0) {
```

Comment:
Should it check for `null` and empty array? Why was it `null` before?

Reply:
I am not sure why it was `null` someone made it. but `getAllIssueComments` returns an array of comments so no need to make the return object `null`,  empty array is better to handle.

Reply:
According to git blame it was @Steveantor. 

Can you explain why you put `null` here?

Reply:
```suggestion
  if (IssueComments?.length === 0) {
```

This also solves for both `null` and empty array.

Reply:
https://github.com/ubiquity/ubiquibot/pull/344#issuecomment-1567720041